jdaviz/core/launcher.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="mx-4">
+    <span style="float: right; font-weight: 100; color: white">
+        <a :href="'https://jdaviz.readthedocs.io/en/'+vdocs" target="__blank" style="color: white">
+            <b>Learn More</b>
+        </a>
+        |
+        <a :href="'https://spacetelescope.github.io/jdat_notebooks/'" target="__blank" style="color: white">
+            <b>Notebooks</b>
+        </a>
+        |
+        <a :href="'https://github.com/spacetelescope/jdaviz'" target="__blank" style="color: white">
+            <b>GitHub</b>
+        </a>
+    </span>
+
+    <h2 class="my-2" style="color: white">Welcome to Jdaviz!</h1>
+    
+    <v-row>
+        <v-text-field
+            v-model="filepath"
+            class="my-4"
+            dark
+            outlined
+            label="File Path"
+            :hint="hint"
+            persistent-hint
+            :loading="hint === 'Please wait. Identifying file...' ? '#C75109' : 'false' " 
+        >
+        </v-text-field>
+
+        <v-dialog v-model="file_chooser_visible" height="400" width="600">
+            <template v-slot:activator="{ on }">
+            <v-btn
+                v-on="on"
+                class="ma-2"
+                style="top: 8px; height: 56px"
+                outlined
+                dark>
+                <v-icon>mdi-file-upload</v-icon
+                <g-file-import id="file-chooser"></g-file-import>
+            </v-btn>
```

Comment:
this would benefit from a tooltip since the icon might otherwise suggest uploading the selected file.

Reply:
Done in https://github.com/spacetelescope/jdaviz/pull/2311/commits/81f2025d82528e21d0a0bb5c6f9eb8279f47ae02